### PR TITLE
Rename packaging

### PR DESCRIPTION
**Pull request by lukewrites**
Change name from `packaging` to `connector-packager`. The python library itself is `connector_packager`, since Python doesn't like using a `-` in library names.

Also updated README.

### CODE REVIEW

File: connector-packager/tests/test_jar_signer.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import unittest
```
Comment (jsun28) [inline connector-packager/tests/test_jar_signer.py]: Is this test_jar_signer.py already in the packaging branch?

Comment (lukewrites) [reply connector-packager/tests/test_jar_signer.py]: looks like it isn't did I pull in something new?

Comment (rosswbrown) [reply connector-packager/tests/test_jar_signer.py]: Ensure you've got 4549b1a

Comment (lukewrites) [reply connector-packager/tests/test_jar_signer.py]: My mistake, it's in there. I think it's showing up as "new" in this PR because it's been moved.

**Comment by lukewrites** [comment]
Figured it out; git thinks the original test file was deleted because there number of lines are different.

**Pull request merged**